src/main/resources/static/assets/css/style.css

```diff
@@ -21673,7 +21673,6 @@ a.material-icons-shopping_cart:hover {
   display: block;
   position: absolute;
   text-align: left;
-  margin-top: 2.3125rem;
```

Comment:
There is a `custom.css` file next to `style.css`, it is already appended to page header in `AbstractFrontendPage.html`, I think it's the best place to add this modification without risking that it could be overwritten during template update.